--- setup.py.old	2023-04-14 17:13:07.566543374 +0200
+++ setup.py	2023-04-14 17:13:38.498470181 +0200
@@ -3,6 +3,12 @@
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
+
+class BinaryDistribution(setuptools.Distribution):
+    def has_ext_modules(foo):
+        return True
+
+
 setuptools.setup(
     name="ghid",
     version="0.1.0",
@@ -19,4 +25,5 @@
         "Operating System :: OS Independent",
     ],
     include_package_data=True,
+    distclass=BinaryDistribution,
 )
